includes/data-stores/interfaces/wc-customer-data-store-interface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC Customer Data Store Interface
+ *
+ * Functions that must be defined by customer store classes.
+ *
+ * @version  2.7.0
+ * @category Interface
+ * @author   WooThemes
+ */
+interface WC_Customer_Data_Store_Interface {
+
+	/**
+	 * Deletes a customer from the database.
+	 *
+	 * @param WC_Customer
+	 * @param int|null $reassign Who to reassign posts to.
+	 */
+	public function delete( &$data, $reassign = null );
```

Comment:
Let's refactor all to accept an array of args, then it can work across all future types.